### PR TITLE
feat(create): add `flame_game` template

### PR DESCRIPTION
**Pull request by wolfenrain**
## Description

Add the `flame_game` template

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by renancaraujo — APPROVED** [review]

**Comment by scarletteliza** [comment]
but where are the docs :P I can handle those

File: test/src/commands/create/create_test.dart [hunk 1]
```diff
@@ -33,6 +33,7 @@ const expectedUsage = [
       '''          [dart_cli]          Generate a Very Good Dart CLI application.\n'''
       '          [dart_pkg]          Generate a reusable Dart package.\n'
       '          [docs_site]         Generate a Very Good documentation site.\n'
+      '          [flame_game]        Generate a Very Good Flame Game.\n'
```
Comment (scarletteliza) [inline test/src/commands/create/create_test.dart:36]:
```suggestion
      '          [flame_game]        Generate a Very Good Flame game.\n'
```

File: test/src/commands/create/create_test.dart [hunk 2]
```diff
@@ -803,6 +804,14 @@ void main() {
             expectedLogSummary: 'Created a Very Good documentation site! 🦄',
           );
         });
+
+        test('flame game template', () async {
+          await expectValidTemplateName(
+            templateName: 'flame_game',
+            expectedBundle: veryGoodFlameGameBundle,
+            expectedLogSummary: 'Created a Very Good Flame Game! 🔥🦄',
```
Comment (scarletteliza) [inline test/src/commands/create/create_test.dart:812]:
```suggestion
            expectedLogSummary: 'Created a Very Good Flame game! 🔥🦄',
```

File: lib/src/commands/create/templates/very_good_flame_game/very_good_flame_game_template.dart [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+/// {@template flame_game_template}
+/// A Flame Game template.
+/// {@endtemplate}
+class VeryGoodFlameGameTemplate extends Template {
+  /// {@macro flame_game_template}
+  VeryGoodFlameGameTemplate()
+      : super(
+          name: 'flame_game',
+          bundle: veryGoodFlameGameBundle,
+          help: 'Generate a Very Good Flame Game.',
```
Comment (wolfenrain) [inline lib/src/commands/create/templates/very_good_flame_game/very_good_flame_game_template.dart:16]:
```suggestion
          help: 'Generate a Very Good Flame game.',
```

File: test/src/commands/create/create_test.dart (hunk 2 above)
Comment (wolfenrain) [reply test/src/commands/create/create_test.dart:812]: No

**Review by felangel — APPROVED** [review]

**Review by felangel — DISMISSED** [review]

File: test/e2e_test.dart [hunk 4]
```diff
@@ -242,6 +242,54 @@ void main() {
...
+
+        final testCoverageResult = await Process.run(
+          'genhtml',
+          ['coverage/lcov.info', '-o', 'coverage'],
+          workingDirectory: directory.path,
+          runInShell: true,
+        );
+        expect(testCoverageResult.exitCode, equals(ExitCode.success.code));
+        expect(testCoverageResult.stderr, isEmpty);
+        expect(testCoverageResult.stdout, contains('lines......: 100.0%'));
```
Comment (felangel) [inline test/e2e_test.dart:290]:
```suggestion
        expect(testCoverageResult.stdout, contains('lines......: 97.8%'));
```

**Review by felangel — APPROVED** [review]

File: test/e2e_test.dart [hunk 5]
```diff
@@ -242,6 +242,54 @@ void main() {
...
+
+        final testCoverageResult = await Process.run(
+          'genhtml',
+          ['coverage/lcov.info', '-o', 'coverage'],
+          workingDirectory: directory.path,
+          runInShell: true,
+        );
+        expect(testCoverageResult.exitCode, equals(ExitCode.success.code));
+        expect(testCoverageResult.stderr, isEmpty);
+        expect(testCoverageResult.stdout, contains('lines......: 97.8%'));
```
Comment (felangel) [inline test/e2e_test.dart:290]: Generated files aren't currently excluded from coverage. Will update this in a subsequent PR

**Pull request merged**